warden/group/manager_sql.go

```diff
@@ -57,16 +57,20 @@ func (m *SQLManager) CreateSchemas() (int, error) {
 	return n, nil
 }
 
+func (m *SQLManager) createGroup(group string) func(tx *sqlx.Tx) error {
+	return func(tx *sqlx.Tx) error {
+		_, err := tx.Exec(m.DB.Rebind("INSERT INTO hydra_warden_group (id) VALUES (?)"), group)
+
+		return errors.WithStack(err)
```

Comment:
Not sure if this works for nil values? So if error is nil, does this return nil as well or not?

Reply:
https://godoc.org/github.com/pkg/errors#WithStack
I looked it up it works as expected

Reply:
Absolutely, it returns nil in that case: https://github.com/pkg/errors/blob/master/errors.go#L145